CHANGELOG.md

```diff
@@ -74,6 +74,8 @@
   - aligned notification page search field color with other pages [#1178](https://github.com/eclipse-tractusx/portal-frontend/issues/1178)
 - **Home**
   - disabled My Apps button for users without permission [#1165](https://github.com/eclipse-tractusx/portal-frontend/issues/1165)
+- **Application Request**
+  - fixed endurl issue in the retriggering of clearing house step [#1268] (https://github.com/eclipse-tractusx/portal-frontend/pull/1268)
```

Comment:
the section wrongly located, in main there's 2.3.0-alpha.3 already released. Please pull the latest state and adjust. Please also put it under Change, according to the PR title and description I just updated.

Reply:
updated

Reply:
only partially addressed